testing/FontIcons/jest.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    'globals': {
+        'ts-jest': {
+            tsConfig: './tsconfig.json',
+        },
+    },
+    moduleFileExtensions: [
+        'ts',
+        'js'
+    ],
+    transform: {
+        '^.+\\.(ts|tsx)$': 'ts-jest',
+    },
+    testEnvironment: 'node',
+    testMatch: [
+        '**/*.test.ts',
+    ],
+    coverageThreshold: {
```

Comment:
We do not need coverage. There is no code for testing. So we do not need to ignore additional folder https://github.com/DevExpress/DevExtreme/pull/14992/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R9